docs/building-from-source.md

```diff
@@ -61,11 +63,13 @@ If you prefer you can check out and patch the repos manually and use those.
 If you check out repos manually then it is your responsibility to ensure that the correct revisions are checked out - see `versions.json` to identify these.
 
 ```
+export CC=clang
+export CXX=clang++
 mkdir repos
-git -C repos clone https://github.com/llvm/llvm-project.git
```

Comment:
For me, the point of specifying an alternate place for the repos is because I want to develop on the code base. And so I want to have the git history.

Otherwise you can just not specify it, and it will be downloaded for you without the history by CMake.

Reply:
Readme file is called building-from-source, and it will be used more often only for build, rather than for developing. One who wants to develop is able to perform the same command manually, but without "--depth=1". After all, the development is much rarer than building

Reply:
Ok, I removed the "--depth=1"